x-pack/plugins/features/server/oss_features.ts

```diff
@@ -555,10 +555,12 @@ export const buildOSSFeatures = ({
             read: [],
           },
           ui: ['saveQuery'],
-        }, // No read-only mode supported
+        },
+        // No read-only mode supported
+        read: { disabled: true, savedObject: { all: [], read: [] }, ui: [] },
```

Comment:
__note:__ it was a bug hidden by `as KibanaFeatureConfig[]` below — the types don't allow omitting the `read` privilege, which could lead to all sorts of runtime bugs. Unnecessary privilege should be disabled instead.